users/api/tests.py

```diff
@@ -0,0 +1,92 @@
+"""
+.. See the NOTICE file distributed with this work for additional information
+   regarding copyright ownership.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+"""
+
+import pytest
+from django.urls import reverse
+from rest_framework.authtoken.models import Token
+
+
+@pytest.fixture
+def api_client():
+    from rest_framework.test import APIClient
+    return APIClient()
+
+
+@pytest.mark.django_db
+def test_login(api_client, django_user_model):
+    """
+    Test user login
+    :param api_client: the API client
+    :param django_user_model: a shortcut to the User model configured for use by the
+    current Django project
+    """
+    username = 'user'
+    password = 'password'
+    django_user_model.objects.create_user(username=username, password=password)
+    url = reverse('login_api')
+    data = {
+        'username': username,
+        'password': password
+    }
+    response = api_client.post(url, data=data)
+    assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_logout(api_client, django_user_model):
+    """
+    Log the user in and log him/her out while providing the access token
```

Comment:
Since we're in inclusive times, perhaps we should avoid gender-specific pronouns wherever possible, as you are excluding the 'they's.